Core/GDCore/IDE/WholeProjectRefactorer.cpp

```diff
@@ -1441,7 +1441,7 @@ void WholeProjectRefactorer::ObjectOrGroupRemovedInLayout(
   }
 }
 
-void WholeProjectRefactorer::ObjectOrGroupRenamedInLayout(
+void WholeProjectRefactorer::RenameObjectOrGroupInLayout(
```

Comment:
The name was like this because this method don't actually rename an object, but just do the refactoring part.